File: modulo1/segmento3/TrabajosPracticos/Apunte1/ejercicio4/ejercicio4.java/Ejercicio4.java
import java.util.Scanner;

public class Ejercicio4 {
	public static void main(String args[]) {	
		Scanner keyboard = new Scanner(System.in);

		System.out.print( "Ingrese el primer numero: " );
		int num1 = keyboard.nextInt();

		System.out.print( "Ingrese el segundo numero: " );
		int num2 = keyboard.nextInt();

		System.out.print( "Ingrese el tercer numero: " );
		int num3 = keyboard.nextInt();

		System.out.print( "Ingrese el cuarto numero: " );
		int num4 = keyboard.nextInt();


    	if ((num1 == num2) && (num3 != num4)) {
			System.out.print( "El primer numero y el segundo numero son iguales." );
		}

		if ((num3 == num4) && (num1 != num2)) {
			System.out.print( "El tercer numero y el cuarto numero son iguales" );
		}

    	if ((num1 == num2) && (num3 == num4)) {
			System.out.print( "El primer numero y el segundo numero son iguales. El tercer numero y el cuarto numero son iguales" );
		}

		keyboard.close();
	}
}

File: modulo1/segmento3/TrabajosPracticos/Apunte1/ejercicio4/ejercicio4.py
print ("Para realizar la comparacion debe ingresar 4 números")

num1 = int(input("Ingrese el primer numero"))
num2 = int(input("Ingrese el segundo numero"))
num3 = int(input("Ingrese el tercer numero"))
num4 = int(input("Ingrese el cuarto numero"))

if ( (num1 == num2) and (num3 != num4) ):
	print ("El primer numero y el segundo numero son iguales.")

if	( (num3 == num4) and (num1 != num2) ):
	print ("El tercer numero y el cuarto numero son iguales.")
	
if ( (num1 == num2) and (num3 == num4)):
	print ("El primer numero y el segundo numero son iguales. El tercer numero y el cuarto numero son iguales")